### PR TITLE
Follow standard explainer template

### PR DESCRIPTION
**Pull request by jbingham**
To make the explainer more readable, this update follows a standard format.

### CODE REVIEW

File: explainer.md [hunk 1]
```diff
@@ -1,27 +1,59 @@
...
 
+## Authors
+
+*   Jonathan Bingham
+*   Honglin Yu
+
+## Participate
+
+*   [Issue tracker](https://github.com/webmachinelearning/model-loader/issues)
+*   [Web ML Working group](https://www.w3.org/groups/wg/webmachinelearning)
```
Comment (anssiko) [inline explainer.md:18]: This spec is currently in the Community Group.

```suggestion
*   [Web Machine Learning Community Group](https://www.w3.org/groups/cg/webmachinelearning)

```

File: explainer.md [hunk 2]
```diff
@@ -1,27 +1,59 @@
...
+
+## Goals
+
+Like the related [Web NN](https://webmachinelearning.github.io/webnn) proposal, some of the main the goals are:
```
Comment (anssiko) [inline explainer.md:22]: Given WebNN is a WG deliverable, it is preferred to use a term "specification" over "proposal".

```suggestion
Like the related [Web Neural Network API](https://webmachinelearning.github.io/webnn) specification, some of the main the goals are: 
```

File: explainer.md [hunk 3]
```diff
@@ -1,27 +1,59 @@
...
-Unlike the Shape Detection API, the model loader APIs are generic. Application-specific libraries and APIs could be built on 
-top.
+*   [TensorFlow Serving](https://www.tensorflow.org/tfx/serving/serving_basic)
+*   [Clipper](https://github.com/ucbrise/clipper)
+*   [TensorRT Inference Server](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/tensorrtserver)
+*   [MXNet Model Server](https://github.com/awslabs/multi-model-server)
 
-The API does not support training a model, modifying a model, federated learning, or other functionality. Maybe future 
-APIs could address those, if useful.
+These and other  APIs are already widely used by many products and organizations for large volumes of requests.
```
Comment (anssiko) [inline explainer.md:53]: Whitespace nit.

```suggestion
These and other APIs are already widely used by many products and organizations for large volumes of requests.

```

File: explainer.md [hunk 4]
```diff
@@ -140,63 +103,125 @@ Any supported model format must be:
...
+to share common data structures and API surfaces for both types of API in order to keep them in sync.
 
-The existence of widely adopted model serving APIs like TensorFlow Serving suggests that an API at this level of abstraction 
-is broadly useful and can be standardized.
+## Application-specific APIs
+
+Application-specific APIs like face detection and barcode detection are also a complementary approach. They have some
+limitations:
+
+Developers want to run custom models too. Application-specific APIs can cover the top use cases only, and with limited flexibility
```
Comment (anssiko) [inline explainer.md:181]: Nit, trailing "."
```suggestion
Developers want to run custom models too. Application-specific APIs can cover the top use cases only, and with limited flexibility.
```

File: explainer.md [hunk 5]
```diff
@@ -140,63 +103,125 @@ Any supported model format must be:
...
+
+In order to potentially support multiple model formats, we considered creating a distinct MLModelLoader class for
+TFLiteModelLoader, to allow the method signature to support TF Lite specific attributes and be more friendly to IDEs.
+Given that our goal is to support a common web standard format, and the method signature for common model formats like
+TFLite and ONNX can be unified, we opted to stay with a more general class.
+
+## Stakeholder feedback / opposition
+
+The Model Loader API is currently included within the tentative scope of the
+[proposed charter]https://www.w3.org/2021/02/proposed-machine-learning-charter.html() for the Web ML Working Group.
```
Comment (anssiko) [inline explainer.md:201]:
```suggestion
The Model Loader API is currently included as a Tentative Specification in the
[Web Machine Learning Working Group Charter](https://www.w3.org/2021/04/web-machine-learning-charter.html).
```

File: explainer.md [hunk 6]
```diff
@@ -140,63 +103,125 @@ Any supported model format must be:
...
+
+*   Apple: No public signals.  
+*   Google: Positive
+*   Intel : Collaborating in Working Group
+*   Microsoft: Collaborating in Working Group
+*   Mozilla: No public signals
+*  Salesforce: Collaborating in Working Group
```
Comment (anssiko) [inline explainer.md:208]: Whitespace nit:
```suggestion
*   Salesforce: Collaborating in Working Group
```

**Review by anssiko — CHANGES_REQUESTED** [review]
Thanks for moving this to the standard format!

**Comment by jbingham** [comment]
I think I accepted all of your changes, @anssiko. It's showing up as Outdated, rather than accepted though. Sorry about that! Can you confirm?

**Review by anssiko — APPROVED** [review]

**Comment by anssiko** [comment]
Yes, all good. 👍

**Review by huningxin — COMMENTED** [review]
LGTM

**Pull request merged**